src/framework/vst/internal/synth/vstsynthesiser.cpp

```diff
@@ -0,0 +1,163 @@
+#include "vstsynthesiser.h"
+
+#include "log.h"
+
+#include "internal/vstplugin.h"
+
+using namespace mu;
+using namespace mu::vst;
+
+VstSynthesiser::VstSynthesiser(const VstPluginMeta& meta)
+    : m_pluginMeta(meta), m_vstAudioClient(std::make_unique<VstAudioClient>())
+{
+}
+
+Ret VstSynthesiser::init()
+{
+    RetVal plugin = repository()->findPluginById(m_pluginMeta.id);
+
+    if (!plugin.ret) {
+        return plugin.ret;
+    }
+
+    m_pluginPtr = plugin.val;
+
+    m_vstAudioClient->init(m_pluginPtr->component());
+
+    return make_ret(Ret::Code::Ok);
+}
+
+bool VstSynthesiser::isValid() const
+{
+    if (!m_pluginPtr) {
+        return false;
+    }
+
+    return m_pluginPtr->isValid();
+}
+
+bool VstSynthesiser::isActive() const
+{
+    return m_isActive;
+}
+
+void VstSynthesiser::setIsActive(bool arg)
+{
+    m_isActive = arg;
+}
+
+std::string VstSynthesiser::name() const
+{
+    return m_pluginMeta.name;
+}
+
+audio::synth::SoundFontFormats VstSynthesiser::soundFontFormats() const
+{
+    NOT_SUPPORTED;
+    return { audio::synth::SoundFontFormat::Undefined };

```

Comment:
Maybe better add new value, like `SoundFontFormat::Embeded` or `SoundFontFormat::External`

Reply:
Embeded sounds good to me